tools/cli/commands/create.py

```diff
@@ -106,10 +106,29 @@
   fi
 }}
 
+checked_format_disk() {{
+  echo "Checking if the persistent disk needs to be formatted"
+  if [ -z "$(blkid ${{PERSISTENT_DISK_DEV}})" ]; then
+    format_disk
+  else
+    echo "Disk already formatted, but mounting failed"
```

Comment:
I think when you reboot, it might be helpful to output a message to that effect.

Reply:
Done.